src/optimisation/linear-problem-data-impl/include/antares/optimisation/linear-problem-data-impl/dataSeries.h

```diff
@@ -15,6 +16,7 @@ class IDataSeries
     }
 
     virtual double getData(unsigned int rank, unsigned int hour) = 0;
+    virtual std::vector<double> getData(unsigned int rank) = 0;
```

Comment:
Is it possible to avoid copies here ?
```suggestion
    virtual const std::vector<double>& getData(unsigned int rank) = 0;
```

Reply:
I've removed this function to avoid checking the consistency between the requested data and the vector size.